From 827e69fa88452ae864db1bc9e700f2a559c3d2d0 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Thu, 29 Jan 2015 17:38:16 -0500
Subject: [misc] redhat: don't suppress Revert patches from changelog

Message-id: <1422553096-52810-1-git-send-email-jarod@redhat.com>
Patchwork-id: 103890
O-Subject: [RHEL7 PATCH] [misc] redhat: don't suppress Revert patches from changelog
Bugzilla: 1187353
RH-Acked-by: Phillip Lougher <plougher@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>

We've been unwittingly suppressing patches with "Revert" in their subjects
from showing up in the changelog for some time now. Back in 2009, prior to
the release of RHEL6, patches that had names starting with ^Revert were
suppressed, and in the migration of the infrastructure to RHEL7, the ^ was
lost and the suppression made much greedier. This removes that suppression
(which was intended to filter out Fedora patches as we rebased pre-RHEL6
on whatever the current Fedora release was) and restores all the changelog
entries with Revert in them since 7.1 started up.

While we're at it, since the next build should be RC, we need to flip
RELEASED_KERNEL to 1, so that the build uses the production UEFI
secureboot keys instead of the beta ones.

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index b544795..2241f6b 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -18,7 +18,7 @@ SOURCES:=$(RPM)/SOURCES
 TESTPATCH:=$(REDHAT)/linux-kernel-test.patch
 FILTERDIFF:=/usr/bin/filterdiff -x '*redhat/*' -x '*/.gitignore' -x '*/makefile' -x '*/Makefile'
 
-RELEASED_KERNEL := 0
+RELEASED_KERNEL := 1
 STAMP_VERSION := $(KVERSION)
 
 LOCVERFILE:=../localversion
diff --git a/redhat/genspec.sh b/redhat/genspec.sh
index db4136c..68ca920 100755
--- a/redhat/genspec.sh
+++ b/redhat/genspec.sh
@@ -131,7 +131,6 @@ cp $clogf.stripped $clogf
 
 if [ "x$HIDE_REDHAT" == "x1" ]; then
  cat $clogf | grep -v -e "^- \[redhat\]" |
-  grep -v "Revert" |
   sed -e 's!\[Fedora\]!!g' > $clogf.stripped
  cp $clogf.stripped $clogf
 fi
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 7465523..cc539d7 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1496,21 +1496,29 @@ fi
 - [fs] ext4: fix overwrite race condition (Jacob Tanenbaum) [1152607] {CVE-2014-8086}
 - [media] ttusb-dec: buffer overflow in ioctl (Alexander Gordeev) [1167116] {CVE-2014-8884}
 - [drm] i915: demote opregion excessive timeout WARN_ONCE to DRM_INFO_ONCE (Rob Clark) [1145627]
+- [md] Revert: raid56: Don't perform reads to support writes until stripe is ready (Jes Sorensen) [1153796]
+- [md] Revert: raid5: avoid livelock caused by non-aligned writes (Jes Sorensen) [1153796]
 - [drm] i915: further quiet i915 (Rob Clark) [1163074]
 - [scsi] megaraid_sas: endianness related bug fixes and code optimization (Tomas Henzl) [1179748]
 - [s390] crypto: kernel oops at insmod of the z90crypt device driver (Hendrik Brueckner) [1172136]
 - [drm] mgag200: Add command line option to specify preferred depth (Dave Airlie) [1044555]
 - [drm] mgag200: Consolidate depth/bpp handling (Dave Airlie) [1044555]
+- [fs] Revert: ext4: revert Disable punch hole on non-extent mapped files (Lukas Czerner) [1176840]
 
 * Sun Jan 25 2015 Jarod Wilson <jarod@redhat.com> [3.10.0-226.el7]
 - [md] dm-cache: fix problematic dual use of a single migration count variable (Mike Snitzer) [1182665]
 - [md] dm-cache: share cache-metadata object across inactive and active DM tables (Mike Snitzer) [1182665]
 - [net] tun/macvtap: use consume_skb() instead of kfree_skb() when needed (Jiri Pirko) [1182805]
+- [virt] Revert: hyperv: Add handler for RNDIS_STATUS_NETWORK_CHANGE event (Jason Wang) [1164163]
 - [virt] kvm/vmx: invalid host cr4 handling across vm entries (Jacob Tanenbaum) [1153329] {CVE-2014-3690}
 - [virt] virtio-scsi: Fix the race condition in virtscsi_handle_event (Fam Zheng) [1152140]
 - [virt] kvm: workaround SuSE's 2.6.16 pvclock vs masterclock issue (Marcelo Tosatti) [1177718]
 - [fs] bdi: avoid oops on device removal (Fam Zheng) [1087179]
 - [mm] backing_dev: fix hung task on sync (Fam Zheng) [1087179]
+- [mm] Revert: vmstat: create separate function to fold per cpu diffs into local counters (Larry Woodman) [1179654]
+- [mm] Revert: vmstat: create fold_diff (Larry Woodman) [1179654]
+- [mm] Revert: vmstat: use this_cpu() to avoid irqon/off sequence in refresh_cpu_vm_stats (Larry Woodman) [1179654]
+- [mm] Revert: vmstat: on-demand vmstat workers V8 (Larry Woodman) [1179654]
 
 * Thu Jan 22 2015 Jarod Wilson <jarod@redhat.com> [3.10.0-225.el7]
 - [net] team: avoid possible underflow of count_pending value for notify_peers and mcast_rejoin (Jiri Pirko) [1176697]
@@ -1601,6 +1609,7 @@ fi
 - [ethernet] enic: fix rx skb checksum (Stefan Assmann) [1154182]
 - [x86] uv: make kdump default action for 'power nmi' (George Beshers) [1175560]
 - [virt] powerpc/kvm: book3s_hv - Fix KSM memory corruption (David Gibson) [1170394]
+- [pci] Revert: Remove from bus_list and release resources in pci_release_dev() (Prarit Bhargava) [1172946]
 - [powercap] rapl: add support for CPU model 0x3f (Rui Wang) [1177579]
 - [kernel] audit: don't attempt to lookup PIDs when changing PID filtering audit rules (Paul Moore) [1172624]
 - [ethernet] ixgbe: avoid possible read_reg panic caused by late method binding (John Greene) [1145772]
@@ -1806,22 +1815,26 @@ fi
 - [wireless] mac80211: schedule the actual switch of the station before CSA count 0 (Stanislaw Gruszka) [1155538]
 - [wireless] mac80211: use secondary channel offset IE also beacons during CSA (Stanislaw Gruszka) [1155538]
 - [wireless] rt2x00: add new rt2800usb device (Stanislaw Gruszka) [1155538]
+- [wireless] Revert: iwlwifi/mvm: treat EAPOLs like mgmt frames wrt rate (Stanislaw Gruszka) [1155538]
 - [wireless] iwlwifi/dvm: drop non VO frames when flushing (Stanislaw Gruszka) [1155538]
 - [wireless] iwlwifi: configure the LTR (Stanislaw Gruszka) [1155538]
 - [wireless] mac80211: fix typo in starting baserate for rts_cts_rate_idx (Stanislaw Gruszka) [1155538]
 - [wireless] rt2x00: add new rt2800usb devices (Stanislaw Gruszka) [1155538]
 - [wireless] rt2x00: support Ralink 5362 (Stanislaw Gruszka) [1155538]
+- [wireless] Revert: ath9k: reduce ANI firstep range for older chips (Stanislaw Gruszka) [1155538]
 - [wireless] rt2800: correct BBP1_TX_POWER_CTRL mask (Stanislaw Gruszka) [1155538]
 - [wireless] iwlwifi: Add missing PCI IDs for the 7260 series (Stanislaw Gruszka) [1155538]
 - [wireless] iwlwifi/mvm: disable BT Co-running by default (Stanislaw Gruszka) [1155538]
 - [wireless] nl80211: clear skb cb before passing to netlink (Stanislaw Gruszka) [1155538]
 - [wireless] ath9k/htc: fix random decryption failure (Stanislaw Gruszka) [1155538]
 - [wireless] brcmfmac: handle IF event for P2P_DEVICE interface (Stanislaw Gruszka) [1155538]
+- [wireless] Revert: mac80211: disable uAPSD if all ACs are under ACM (Stanislaw Gruszka) [1155538]
 - [wireless] rtlwifi/rtl8192cu: Add new ID (Stanislaw Gruszka) [1155538]
 - [wireless] iwlwifi/mvm: set MAC_FILTER_IN_BEACON correctly for STA/P2P client (Stanislaw Gruszka) [1155538]
 - [wireless] iwlwifi/mvm: treat EAPOLs like mgmt frames wrt rate (Stanislaw Gruszka) [1155538]
 - [wireless] iwlwifi: increase DEFAULT_MAX_TX_POWER (Stanislaw Gruszka) [1155538]
 - [wireless] iwlwifi/mvm: fix endianity issues with Smart Fifo commands (Stanislaw Gruszka) [1155538]
+- [wireless] Revert: iwlwifi/dvm: don't enable CTS to self (Stanislaw Gruszka) [1155538]
 - [wireless] carl9170: fix sending URBs with wrong type when using full-speed (Stanislaw Gruszka) [1155538]
 
 * Fri Dec 12 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-217.el7]
@@ -1857,6 +1870,7 @@ fi
 - [x86] traps: stop using IST for #SS (Petr  Matousek) [1172813] {CVE-2014-9322}
 
 * Tue Dec 09 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-216.el7]
+- [acpi] Revert: hotplug/pci: Simplify disable_slot() (Prarit Bhargava) [1158720]
 - [infiniband] iser: Adjust data_length to include protection information (Amir Vadai) [1107622]
 - [infiniband] iser: Bump version to 1.4.1 (Amir Vadai) [1107622]
 - [infiniband] iser: Allow bind only when connection state is UP (Amir Vadai) [1107622]
@@ -2349,12 +2363,14 @@ fi
 - [ethernet] i40e: disable FCoE (Stefan Assmann) [1165175]
 - [cpufreq] intel_pstate: Add CPUID for BDW-H CPU (Steve Best) [1164379]
 - [mm] do not overwrite reserved pages counter at show_mem() (Rafael Aquini) [1125433]
+- [alsa] Revert: Kconfig: rename HAS_IOPORT to HAS_IOPORT_MAP (Jarod Wilson) [1112200]
 - [ethernet] enic: Do not call napi_disable when preemption is disabled (Stefan Assmann) [1145019]
 - [ethernet] enic: fix possible deadlock in enic_stop/ enic_rfs_flw_tbl_free (Stefan Assmann) [1145019]
 - [x86] uv_bau: Avoid NULL pointer reference in ptc_seq_show (Frank Ramsay) [1161183]
 - [x86] uv_bau: Increase maximum CPUs per socket/hub (Frank Ramsay) [1161183]
 - [mm] vmscan: do not throttle based on pfmemalloc reserves if node has no ZONE_NORMAL (Gustavo Duarte) [1148925]
 - [char] hwrng/pseries: port to new read API and fix stack corruption (Gustavo Duarte) [1163659]
+- [md] Revert: dm-cache: add call to mark_tech_preview (Mike Snitzer) [1159001]
 - [md] dm-cache: emit a warning message if there are a lot of cache blocks (Mike Snitzer) [1159001]
 - [md] dm-cache: improve discard support (Mike Snitzer) [1159001]
 - [md] dm-cache: revert "prevent corruption caused by discard_block_size > cache_block_size" (Mike Snitzer) [1159001]
@@ -2789,7 +2805,9 @@ fi
 - [fs] ext4: remove unused discard_partial_page_buffers (Lukas Czerner) [1144326]
 - [fs] ext4: use ext4_zero_partial_blocks in punch_hole (Lukas Czerner) [1144326]
 - [fs] ext4: truncate_inode_pages() in orphan cleanup path (Lukas Czerner) [1144326]
+- [fs] Revert "ext4: fix fsx truncate failure" (Lukas Czerner) [1144326]
 - [fs] ext4: Call ext4_jbd2_file_inode() after zeroing block (Lukas Czerner) [1144326]
+- [fs] Revert "ext4: remove no longer used functions in inode.c" (Lukas Czerner) [1144326]
 - [fs] mm: teach truncate_inode_pages_range() to handle non page aligned ranges (Lukas Czerner) [1144326]
 - [fs] ext4: use ->invalidatepage() length argument (Lukas Czerner) [1144326]
 - [fs] jbd2: change jbd2_journal_invalidatepage to accept length (Lukas Czerner) [1144326]
@@ -3096,6 +3114,7 @@ fi
 - [mm] thp: do not allow thp faults to avoid cpuset restrictions (Frank Ramsay) [1149786]
 - [mm] thp: add VM_INIT_DEF_MASK and PRCTL_THP_DISABLE (Frank Ramsay) [1149786]
 - [fs] exec: kill the unnecessary mm->def_flags setting in load_elf_binary() (Frank Ramsay) [1149786]
+- [mm] Revert: thp: make MADV_HUGEPAGE check for mm->def_flags (Frank Ramsay) [1149786]
 - [block] disable entropy contributions for nonrot devices (Mike Snitzer) [852425]
 - [crypto] qat: Enforce valid numa configuration (Nikolay Aleksandrov) [1152642]
 - [crypto] qat: Prevent dma mapping zero length assoc data (Nikolay Aleksandrov) [1152642]
@@ -3132,6 +3151,7 @@ fi
 - [fs] fuse: Timeout comparison fix (Carlos Maiolino) [1128802]
 - [fs] lockd: Try to reconnect if statd has moved (Benjamin Coddington) [1120850]
 - [fs] sunrpc: Don't wake tasks during connection abort (Benjamin Coddington) [1120850]
+- [fs] Revert: writeback: do not sync data dirtied after sync start (Brian Foster [1085828]
 - [fs] nfsd4: fix corruption of NFSv4 read data ("J. Bruce Fields") [1148055]
 - [fs] gfs2: revert "GFS2: d_splice_alias() can't return error" (Robert S Peterson) [1141899]
 
@@ -3470,6 +3490,10 @@ fi
 - [kernel] sched/balancing: Reduce the rate of needless idle load balancing (Larry Woodman) [1120318]
 - [kernel] sched/fair: Implement fast idling of CPUs when the system is partially loaded (Larry Woodman) [1120318]
 - [kernel] sched: Skip double execution of pick_next_task_fair() (Larry Woodman) [1120318]
+- [md] Revert: dm: change sector_count member in clone_info from sector_t to unsigned (Mike Snitzer) [1147574]
+- [md] Revert: dm: introduce dm_accept_partial_bio (Mike Snitzer) [1147574]
+- [md] Revert: dm-snapshot: allocate a per-target structure for snapshot-origin target (Mike Snitzer) [1147574]
+- [md] Revert: dm-snapshot: do not split read bios sent to snapshot-origin target (Mike Snitzer) [1147574]
 - [virt] kvm: nested vmx - disable perf cpuid reporting (Marcelo Tosatti) [1083969]
 - [mm] slub: Handle NULL parameter in kmem_cache_flags (Ilya Dryomov) [1143028]
 - [crypto] algif: avoid excessive use of socket buffer in skcipher (Ondrej Kozina) [1136075]
@@ -4076,6 +4100,7 @@ fi
 - [tools] perf/lib/traceevent: Add options to plugins (Jiri Olsa) [1134356]
 - [tools] perf/lib/traceevent: Add flag to not load event plugins (Jiri Olsa) [1134356]
 - [tools] perf/documentation: Add description for conditional branch filter (Jiri Olsa) [1134356]
+- [tools] Revert: perf: Disable PERF_RECORD_MMAP2 support (Jiri Olsa) [1134356]
 - [tools] perf: Add conditional branch filter 'cond' to perf record (Jiri Olsa) [1134356]
 - [tools] perf: Fix 'make help' message error (Jiri Olsa) [1134356]
 - [tools] perf/record: Fix poll return value propagation (Jiri Olsa) [1134356]
@@ -4309,6 +4334,7 @@ fi
 - [x86] perf/intel: Protect LBR and extra_regs against KVM lying (Jiri Olsa) [1134356]
 - [kernel] perf: Fix lockdep warning on process exit (Jiri Olsa) [1134356]
 - [x86] perf/intel: Use proper dTLB-load-misses event on IvyBridge (Jiri Olsa) [1134356]
+- [kernel] Revert: perf: Always destroy groups on exit (Jiri Olsa) [1134356]
 - [kernel] perf: Do not allow optimized switch for non-cloned events (Jiri Olsa) [1134356]
 - [kernel] perf: Pass protection and flags bits through mmap2 interface (Jiri Olsa) [1134356]
 - [kernel] perf: Differentiate exec() and non-exec() comm events (Jiri Olsa) [1134356]
@@ -4683,6 +4709,7 @@ fi
 - [alsa] hda: Add the pin fixup for HP Envy TS bass speaker (Jaroslav Kysela) [1112200]
 - [alsa] control: Define SNDRV_CTL_TLV_OP_* constants (Jaroslav Kysela) [1112200]
 - [alsa] hda: Fix broken PM due to incomplete i915 initialization (Jaroslav Kysela) [1112200]
+- [alsa] hda: Revert stream assignment order for Intel controllers (Jaroslav Kysela) [1112200]
 - [alsa] pcm: Add timestamp type to sw_params (Jaroslav Kysela) [1112200]
 - [alsa] pcm: simplify snd_pcm_tstamp() (Jaroslav Kysela) [1112200]
 - [alsa] hda: add capture mute led support in led_power_filter (Jaroslav Kysela) [1112200]
@@ -4737,6 +4764,7 @@ fi
 - [alsa] core: Use ktime_get_ts() (Jaroslav Kysela) [1112200]
 - [alsa] hda: verify pin-converter connection on unsol event for HSW and VLV (Jaroslav Kysela) [1112200]
 - [alsa] hda: Add quirk for ABit AA8XE (Jaroslav Kysela) [1112200]
+- [alsa] Revert: hda: mask buggy stream DMA0 for Broadwell display controller (Jaroslav Kysela) [1112200]
 - [alsa] hda: using POS_FIX_LPIB on Broadwell HDMI Audio (Jaroslav Kysela) [1112200]
 - [alsa] hda/realtek: Add support of ALC667 codec (Jaroslav Kysela) [1112200]
 - [alsa] hda/realtek: Add more codec rename (Jaroslav Kysela) [1112200]
@@ -4750,6 +4778,7 @@ fi
 - [alsa] hda/analog: Fix silent output on ASUS A8JN (Jaroslav Kysela) [1112200]
 - [alsa] hda: move some alc662 family machines to hda_pin_quirk table (Jaroslav Kysela) [1112200]
 - [alsa] hda: move some alc269 family machines to hda_pin_quirk table (Jaroslav Kysela) [1112200]
+- [alsa] Revert: hda: drop def association and sequence from pinconf comparing (Jaroslav Kysela) [1112200]
 - [alsa] hda: fix tegra build (Jaroslav Kysela) [1112200]
 - [alsa] hda: Pop noises fix for XPS13 9333 (Jaroslav Kysela) [1112200]
 - [alsa] hda: add an instance to use snd_hda_pick_pin_fixup (Jaroslav Kysela) [1112200]
@@ -5205,6 +5234,18 @@ fi
 - [usb] pl2303: fix data corruption on termios updates (Don Zickus) [1110939]
 - [usb] pl2303: add error handling to set_control_lines (Don Zickus) [1110939]
 - [usb] pl2303: add error handling to vendor read and write functions (Don Zickus) [1110939]
+- [usb] Revert: pl2303: restrict the divisor based baud rate encoding method to the "HX" chip type (Don Zickus) [1110939]
+- [usb] Revert: pl2303: fix+improve the divsor based baud rate encoding method (Don Zickus) [1110939]
+- [usb] Revert: pl2303: do not round to the next nearest standard baud rate for the divisor based baud rate encoding method (Don Zickus) [1110939]
+- [usb] Revert: pl2303: remove 500000 baud from the list of standard baud rates (Don Zickus) [1110939]
+- [usb] Revert: pl2303: move the two baud rate encoding methods to separate functions (Don Zickus) [1110939]
+- [usb] Revert: pl2303: increase the allowed baud rate range for the divisor based encoding method (Don Zickus) [1110939]
+- [usb] Revert: pl2303: also use the divisor based baud rate encoding method for baud rates < 115200 with HX chips (Don Zickus) [1110939]
+- [usb] Revert: pl2303: add two comments concerning the supported baud rates with HX chips (Don Zickus) [1110939]
+- [usb] Revert: pl2303: simplify the else-if contruct for type_1 chips in pl2303_startup() (Don Zickus) [1110939]
+- [usb] Revert: pl2303: improve the chip type information output on startup
+- [usb] Revert: pl2303: improve the chip type detection/distinction
+- [usb] Revert: pl2303: distinguish between original and cloned HX chips
 - [usb] pl2303: clean up driver somewhat (Don Zickus) [1110939]
 - [usb] pl2303: remove bogus delta_msr_wait wake up (Don Zickus) [1110939]
 - [usb] core: let dynamic ids override static ids (Don Zickus) [1110939]
@@ -5349,6 +5390,9 @@ fi
 - [usb] qcserial: fix multiline comment coding style (Don Zickus) [1110939]
 - [usb] Avoid runtime suspend loops for HCDs that can't handle suspend/resume (Don Zickus) [1110939]
 - [usb] xhci: rework command timeout and cancellation, (Don Zickus) [1110939]
+- [usb] Revert: xhci: replace xhci_read_64() with readq() (Don Zickus) [1110939]
+- [usb] Revert: xhci: replace xhci_write_64() with writeq() (Don Zickus) [1110939]
+- [usb] Revert: core: set lpm_capable field for LPM capable root hubs (Don Zickus) [1110939]
 - [usb] xhci: Use completion and status in global command queue (Don Zickus) [1110939]
 - [usb] xhci: Add a global command queue (Don Zickus) [1110939]
 - [usb] xhci: Use command structures when queuing commands on the command ring (Don Zickus) [1110939]
@@ -5392,6 +5436,7 @@ fi
 - [usb] pl2303: add ids for Hewlett-Packard HP POS pole displays (Don Zickus) [1110939]
 - [usb] cp210x: Add 8281 (Nanotec Plug & Drive) (Don Zickus) [1110939]
 - [usb] option driver, add support for Telit UE910v2 (Don Zickus) [1110939]
+- [usb] Revert: serial: add usbid for dell wwan card to sierra.c (Don Zickus) [1110939]
 - [usb] serial/ftdi_sio: add id for Brainboxes serial cards (Don Zickus) [1110939]
 - [net] usb: include wait queue head in device structure (Don Zickus) [1110939]
 - [usb] disable reset-resume when USB_QUIRK_RESET is set (Don Zickus) [1110939]
@@ -5403,6 +5448,7 @@ fi
 - [usb] serial: continue to read on errors (Don Zickus) [1110939]
 - [usb] serial: make bulk_out_size a lower limit (Don Zickus) [1110939]
 - [usb] cypress_m8: fix potential scheduling while atomic (Don Zickus) [1110939]
+- [usb] Revert: xhci: Limit arbitrarily-aligned scatter gather (Don Zickus) [1110939]
 - [usb] Make DELAY_INIT quirk wait 100ms between Get Configuration requests (Don Zickus) [1110939]
 - [usb] Add device quirk for Logitech HD Pro Webcams C920 and C930e (Don Zickus) [1110939]
 - [usb] xhci: add the meaningful IRQ description if it is empty (Don Zickus) [1110939]
@@ -5494,6 +5540,9 @@ fi
 - [usb] option: blacklist ZTE MF667 net interface (Don Zickus) [1110939]
 - [usb] misc/usbled: Add Riso Kagaku Webmail Notifier (Don Zickus) [1110939]
 - [usb] elan: Remove useless "default M" lines (Don Zickus) [1110939]
+- [usb] Revert: xhci: Link TRB must not occur within a USB payload burst (Don Zickus) [1110939]
+- [usb] Revert: xhci: Avoid infinite loop when sg urb requires too many trbs (Don Zickus) [1110939]
+- [usb] Revert: xhci: Set scatter-gather limit to avoid failed block writes (Don Zickus) [1110939]
 - [usb] xhci: Limit arbitrarily-aligned scatter gather (Don Zickus) [1110939]
 - [usb] delete non-required instances of include <linux/init.h> (Don Zickus) [1110939]
 - [usb] hub set hub->change_bits when over-current happens (Don Zickus) [1110939]
@@ -5541,6 +5590,7 @@ fi
 - [usb] core: allow isoc URBs for wireless devices with an interval < 6 (Don Zickus) [1110939]
 - [usb] serial/zte_ev: move support for ZTE AC2726 from zte_ev back to option (Don Zickus) [1110939]
 - [usb] cdc-wdm: manage_power should always set needs_remote_wakeup (Don Zickus) [1110939]
+- [usb] Revert: quirks: add touchscreen that is dazzeled by remote wakeup (Don Zickus) [1110939]
 - [usb] Use dev_is_pci() to check whether it is pci device (Don Zickus) [1110939]
 - [usb] usbtest: update bos test coverage to usb 2.1 device (Don Zickus) [1110939]
 - [usb] usbtest: fix the bit mask of usb 2.0 extension descriptor (Don Zickus) [1110939]
@@ -6093,6 +6143,7 @@ fi
 - [acpi] apci_pad: in power_saving_thread(), call schedule() when need_resched is true (Tony Camuso) [1092076]
 
 * Wed Sep 24 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-170.el7]
+- [powerpc] Revert: Add iommu domain pointer to device archdata (David Gibson) [1123145 1123133 1123367]
 - [powerpc] Enable IOMMU and VFIO for ppc64 (David Gibson) [1123145 1123133 1123367]
 - [virt] kvm/ppc: book3s_32/pr - Access HTAB in big endian (David Gibson) [1123145 1123133 1123367]
 - [virt] kvm/ppc: book3s/pr - Rework SLB switching code (David Gibson) [1123145 1123133 1123367]
@@ -6243,6 +6294,7 @@ fi
 - [ethernet] bnx2x: prevent incorrect byte-swap in BE (Michal Schmidt) [1119861 1060671 1089402]
 - [ethernet] bnx2x: Utilize FW 7.10.51 (Michal Schmidt) [1119861 1060671 1089402]
 - [ethernet] bnx2x: Remove redundant #ifdef (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Revert UNDI flushing mechanism (Michal Schmidt) [1089402]
 - [ethernet] bnx2x: Remove DEFINE_PCI_DEVICE_TABLE macro use (Michal Schmidt) [1089402]
 - [ethernet] bnx2x: fix set_setting for some PHYs (Michal Schmidt) [1089402]
 - [ethernet] bnx2x: fix possible panic under memory stress (Michal Schmidt) [1089402]
@@ -6300,6 +6352,7 @@ fi
 - [ethernet] e1000e: Fix TSO with non-accelerated vlans (Vlad Yasevich) [1135619]
 - [net] macvlan: Initialize vlan_features to turn on offload support (Vlad Yasevich) [1135619]
 - [net] macvtap: Fix checksum errors for non-gso packets in bridge mode (Vlad Yasevich) [1135619]
+- [net] Revert: macvlan: fix checksums error when we are in bridge mode (Vlad Yasevich) [1135619]
 - [net] Correctly set segment mac_len in skb_segment() (Vlad Yasevich) [1135619]
 - [crypto] pkcs7: Fix the parser cleanup to drain parsed out X.509 certs (Vivek Goyal) [1129779]
 - [crypto] pkcs7: Provide a single place to do signed info block freeing (Vivek Goyal) [1129779]
@@ -6317,6 +6370,7 @@ fi
 - [crypto] keys: make partial key id matching as a dedicated function (Vivek Goyal) [1129779]
 - [crypto] keys: verify a certificate is signed by a 'trusted' key (Vivek Goyal) [1129779]
 - [security] keys: special dot prefixed keyring name bug fix (Vivek Goyal) [1129779]
+- [crypto] Revert: keys: verify a certificate is signed by a 'trusted' key (Vivek Goyal) [1129779]
 - [crypto] pefile: Validate PKCS#7 trust chain (Vivek Goyal) [1129779]
 - [crypto] pefile: Digest the PE binary and compare to the PKCS#7 data (Vivek Goyal) [1129779]
 - [crypto] pefile: Handle pesign using the wrong OID (Vivek Goyal) [1129779]
@@ -7091,6 +7145,7 @@ fi
 - [powerpc] book3s: Recover from MC in sapphire on SCOM read via MMIO (Don Zickus) [1127366]
 - [powerpc] pseries: Don't try to register pseries cpu hotplug on non-pseries (Don Zickus) [1127366]
 - [powerpc] Fix xmon disassembler for little-endian (Don Zickus) [1127366]
+- [powerpc] Revert c6102609 and replace it with the correct fix for vio dma mask setting (Don Zickus) [1127366]
 - [powerpc] Kill CONFIG_MTD_PARTITIONS (Don Zickus) [1127366]
 - [powerpc] Align p_dyn, p_rela and p_st symbols (Don Zickus) [1127366]
 - [powerpc] powernv: Add OPAL call to resync timebase on wakeup (Don Zickus) [1127366]
@@ -7372,6 +7427,7 @@ fi
 - [powerpc] pseries: Fix a typo in pSeries_lpar_hpte_insert() (Don Zickus) [1127366]
 - [powerpc] Fix the corrupt r3 error during MCE handling (Don Zickus) [1127366]
 - [powerpc] Access local paca after hard irq disabled (Don Zickus) [1127366]
+- [powerpc] Revert: hw_breakpoints: Fix racy access to ptrace breakpoints (Don Zickus) [1127366]
 - [powerpc] kvm: Use 256K chunk to track both RMA and hash page table allocation (Don Zickus) [1127366]
 - [powerpc] kvm: Contiguous memory allocator based RMA allocation (Don Zickus) [1127366]
 - [powerpc] kvm: Contiguous memory allocator based hash page table allocation (Don Zickus) [1127366]
@@ -7537,6 +7593,7 @@ fi
 - [cpufreq] Prevent problems in update_policy_cpu() if last_cpu == new_cpu (Prarit Bhargava) [1134369]
 - [cpufreq] Restructure if/else block to avoid unintended behavior (Prarit Bhargava) [1134369]
 - [cpufreq] Fix crash in cpufreq-stats during suspend/resume (Prarit Bhargava) [1134369]
+- [cpufreq] Revert: make sure frequency transitions are serialized (Prarit Bhargava) [1134369]
 - [cpufreq] Use signed type for 'ret' variable, to store negative error values (Prarit Bhargava) [1134369]
 - [cpufreq] Remove temporary fix for race between CPU hotplug and sysfs-writes (Prarit Bhargava) [1134369]
 - [cpufreq] Synchronize the cpufreq store_*() routines with CPU hotplug (Prarit Bhargava) [1134369]
@@ -9479,6 +9536,7 @@ fi
 - [mm] thp: fix copy_page_rep GPF by testing is_huge_zero_pmd once only (Rik van Riel) [1123040]
 - [mm] numa: Remove BUG_ON() in __handle_mm_fault() (Rik van Riel) [1119439]
 - [iommu] Add empty stub for iommu_group_get_by_id() (Steve Best) [1123122]
+- [input] Revert: wacom: testing result shows get_report is unnecessary (Aristeu Rozanski) [1079541]
 - [kernel] kexec: fix build error when hugetlbfs is disabled (Baoquan He) [1115240]
 - [kernel] kexec: export free_huge_page to VMCOREINFO (Baoquan He) [1115240]
 - [kernel] kexec: save PG_head_mask in VMCOREINFO (Baoquan He) [1115240]
@@ -9561,6 +9619,7 @@ fi
 - [net] sctp: fix a missed .data initialization (Daniel Borkmann) [1110290]
 - [net] sctp: fix up a spacing (Daniel Borkmann) [1110290]
 - [net] sctp: add check rto_min and rto_max in sysctl (Daniel Borkmann) [1110290]
+- [net] Revert: introduce netif_skb_dev_features (Florian Westphal) [1109214]
 - [net] ip: push gso skb forwarding handling down the stack (Florian Westphal) [1109214]
 - [net] ipv6: send pkttoobig immediately if orig frag size > mtu (Florian Westphal) [1109214]
 - [net] ipv4: ip_forward: fix inverted local_df test (Florian Westphal) [1109214]
@@ -9806,6 +9865,7 @@ fi
 - [hwrng] add per-device entropy derating (Amit Shah) [1129195]
 - [hwrng] create filler thread (Amit Shah) [1129195]
 - [hwrng] random: add_hwgenerator_randomness() for feeding entropy from devices (Amit Shah) [1129195]
+- [char] Revert: hwrng: virtio - ensure reads happen after successful probe (Amit Shah) [1129195]
 - [char] hwrng: virtio - delay hwrng_register() till driver is ready (Amit Shah) [1119299]
 - [char] hwrng: virtio - re-arrange struct elements for better packing (Amit Shah) [1119299]
 - [char] hwrng: virtio - remove unused struct element (Amit Shah) [1119299]
@@ -10126,6 +10186,7 @@ fi
 - [fs] btrfs: fix possible deadlock in btrfs_cleanup_transaction (Eric Sandeen) [1071179]
 - [fs] btrfs: faster/more efficient insertion of file extent items (Eric Sandeen) [1071179]
 - [fs] btrfs: always choose work from prio_head first (Eric Sandeen) [1071179]
+- [fs] Revert: btrfs: remove transaction from btrfs send (Eric Sandeen) [1071179]
 - [fs] btrfs: skip readonly root for snapshot-aware defragment (Eric Sandeen) [1071179]
 - [fs] btrfs: switch to btrfs_previous_extent_item() (Eric Sandeen) [1071179]
 - [fs] btrfs: skip submitting barrier for missing device (Eric Sandeen) [1071179]
@@ -10155,6 +10216,7 @@ fi
 - [fs] btrfs: unset DCACHE_DISCONNECTED when mounting default subvol (Eric Sandeen) [1071179]
 - [fs] btrfs: fix max_inline mount option (Eric Sandeen) [1071179]
 - [fs] btrfs: fix a lockdep warning when cleaning up aborted transaction (Eric Sandeen) [1071179]
+- [fs] Revert: btrfs: add ioctl to export size of global metadata reservation (Eric Sandeen) [1071179]
 - [fs] btrfs: fix data corruption when reading/updating compressed extents (Eric Sandeen) [1071179]
 - [fs] btrfs: don't loop forever if we can't run because of the tree mod log (Eric Sandeen) [1071179]
 - [fs] btrfs: reserve no transaction units in btrfs_ioctl_set_features (Eric Sandeen) [1071179]
@@ -10606,6 +10668,7 @@ fi
 - [lib] percpu_counter: fix __percpu_counter_add() (Eric Sandeen) [1123968]
 
 * Fri Aug 01 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-142.el7]
+- [x86] Revert: Insert GART region into resource map (Myron Stowe) [1110895]
 - [pci] Don't check resource_size() in pci_bus_alloc_resource() (Myron Stowe) [1110895]
 - [pci] Enable INTx in pci_reenable_device() only when MSI/MSI-X not enabled (Myron Stowe) [1110895]
 - [ahci] Fix broken fallback to single MSI mode (Myron Stowe) [1110895]
@@ -10616,6 +10679,8 @@ fi
 - [pci] msi: Fix pci_msix_vec_count() htmldocs failure (Myron Stowe) [1110895]
 - [pci] msi: Fix leak of msi_attrs (Myron Stowe) [1110895]
 - [pci] msi: Check kmalloc() return value, fix leak of name (Myron Stowe) [1110895]
+- [eisa] Revert: Initialize device before its resources (Myron Stowe) [1110895]
+- [eisa] Revert: Log device resources in dmesg (Myron Stowe) [1110895]
 - [pci] Check parent kobject in pci_destroy_dev() (Myron Stowe) [1110895]
 - [powerpc] eeh: Use global PCI rescan-remove locking (Myron Stowe) [1110895]
 - [pci] Fix pci_check_and_unmask_intx() comment typos (Myron Stowe) [1110895]
@@ -10798,6 +10863,7 @@ fi
 - [md] dm: remove dm_get_mapinfo (Mike Snitzer) [1117872]
 - [md] dm-table: make dm_table_supports_discards static (Mike Snitzer) [1117872]
 - [md] dm: make dm_table_alloc_md_mempools static (Mike Snitzer) [1117872]
+- [mm] Revert: vmscan: do not swap anon pages just because free+file is low (Johannes Weiner) [1102991]
 - [crypto] fips: only panic on bad/missing crypto mod signatures (Jarod Wilson) [903562]
 - [x86] hugetlb: Add missing TLB page invalidation for hugetlb_cow() (Larry Woodman) [1097919]
 
@@ -10841,6 +10907,7 @@ fi
 - [x86] msr: add 64bit _on_cpu access functions (Steve Best) [1085575]
 - [misc] bitops: Introduce BIT_ULL (Steve Best) [1085575]
 - [virt] xen/spinlock: Don't use __initdate for xen_pv_spin (Vitaly Kuznetsov) [1006977]
+- [virt] Revert: xen/spinlock: Disable IRQ spinlock (PV) allocation on PVHVM (Vitaly Kuznetsov) [1006977]
 - [virt] xen/spinlock: Don't setup xen spinlock IPI kicker if disabled (Vitaly Kuznetsov) [1006977]
 - [virt] xen/smp: Update pv_lock_ops functions before alternative code starts under PVHVM (Vitaly Kuznetsov) [1006977]
 - [virt] xen/spinlock: We don't need the old structure anymore (Vitaly Kuznetsov) [1006977]
@@ -10965,6 +11032,7 @@ fi
 - [scsi] avoid taking host_lock in scsi_run_queue unless nessecary (Mike Snitzer) [1105204]
 - [scsi] avoid useless free_list lock roundtrips (Mike Snitzer) [1105204]
 - [scsi] enable destruction of blocked devices which fail LUN scanning (Mike Snitzer) [1105204]
+- [scsi] Revert: reintroduce scsi_driver.init_command (Mike Snitzer) [1105204]
 - [block] implement an unprep function corresponding directly to prep (Mike Snitzer) [1105204]
 - [scsi] handle command allocation failure in scsi_reset_provider (Mike Snitzer) [1105204]
 - [block] properly stack underlying max_segment_size to DM device (Mike Snitzer) [1105204]
@@ -11046,6 +11114,7 @@ fi
 - [block] fold __blk_add_timer into blk_add_timer (Mike Snitzer) [1105204]
 - [block] blk-mq: respect rq_affinity (Mike Snitzer) [1105204]
 - [block] blk-mq: fix race with timeouts and requeue events (Mike Snitzer) [1105204]
+- [block] Revert: blk-mq: initialize req->q in allocation (Mike Snitzer) [1105204]
 - [block] blk-mq: fix leak of set->tags (Mike Snitzer) [1105204]
 - [block] blk-mq: initialize req->q in allocation (Mike Snitzer) [1105204]
 - [block] blk-mq: user (1 << order) to implement order_to_size() (Mike Snitzer) [1105204]
@@ -11137,6 +11206,7 @@ fi
 - [acpi] hotplug: Refuse to hot-remove all objects with disabled hotplug (Myron Stowe) [1114228]
 - [acpi] scan: Start matching drivers after trying scan handlers (Myron Stowe) [1114228]
 - [acpi] Remove acpi_pci_slot_init() headers from internal.h (Myron Stowe) [1114228]
+- [pci] Revert: acpi / hotplug / pci: Avoid doing too much for spurious notifies (Myron Stowe) [1114228]
 - [base] Release device_hotplug_lock when store_mem_state returns EINVAL (Myron Stowe) [1114228]
 - [acpi] hotplug/pci: Drop WARN_ON() from acpiphp_enumerate_slots() (Myron Stowe) [1114228]
 - [acpi] hotplug/pci: Fix error code path in acpiphp_enumerate_slots() (Myron Stowe) [1114228]
@@ -11572,6 +11642,7 @@ fi
 - [kernel] ftrace: Hardcode ftrace_module_init() call into load_module() (Takahiro MUNEDA) [1061553]
 - [kernel] trace: Make register/unregister_ftrace_command __init (Takahiro MUNEDA) [1061553]
 - [tools] turbostat: run on HSX (Prarit Bhargava) [1081657]
+- [tools] Revert: turbostat: servers do not support uncore power register (Prarit Bhargava) [1081657]
 
 * Thu Jul 03 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-132.el7]
 - [net] neigh: set nud_state to NUD_INCOMPLETE when probing router reachability (Hannes Frederic Sowa) [1110193]
-- 
1.7.1